tests/test_example_dags.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pytest
+from airflow.executors.debug_executor import DebugExecutor
+from airflow.models.dagbag import DagBag
+
+# from astro.sql.operators.temp_hooks import PostgresHook
+from airflow.utils import timezone
+from airflow.utils.session import provide_session
+from airflow.utils.state import State
+from google.api_core.exceptions import NotFound
+
+# Import Operator
+
+DEFAULT_SCHEMA = "tmp_astro"
+DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+
+
+@provide_session
+def get_session(session=None):
+    return session
+
+
+@pytest.fixture()
+def session():
+    return get_session()
+
+
+@pytest.mark.parametrize(
+    "dag_id",
+    [
```

Comment:
Would it be worth listing all the dags in the examples folder  (e.g. `glob.glob("../../example_dags/*.py`) instead of having a hard-coded list?

Reply:
I think we might want to pick and choose instead of running all dags?

Reply:
I believe all the DAGs in our examples folders should be functional - and my understanding is that this is the way we have to make sure they'll be working. My concern with having a hard-coded list here is that we may forget to update this test when new example DAGs are added - and they may stop working if we don't have an automatic way of validating them.

If we want to run specific tests for local development, we can always do that with `pytest -k`

Reply:
I've set the tests to run against all DAGs, but I think having this list of DAG IDs is the easiest way to parameterize. I'm not sure how we would go about getting these DAG ids otherwise.

Reply:
@tatiana I have added all DAGs to the test

Reply:
@tatiana I tried to make it grab all DAGs from the dagbag which would've allowed this to automatically pick up new dags but for some reason that caused the CI to infinitely loop. I suppose we can just make sure that any new dags get added to the list?

Reply:
That's a shame. Please, let me know if you'd like to pair on this at some point - with a rubber duck, we may be able to find a way that works.
It is not a blocker, though. You did great work which deserves to be in our main branch! :tada:

Reply:
Actually, it seems you solved it in this otherPR: